readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://katenevm.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://katenevm.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add this link to Pull Request description
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/b79f7de2-7129-4067-99b2-0234eb72fe8b)